x-pack/plugins/lens/public/visualizations/legacy_metric/visualization.tsx

```diff
@@ -158,7 +159,7 @@ const toExpression = (
           showLabels: [!attributes?.mode || attributes?.mode === 'full'],
           colorMode: !canColor ? [ColorMode.None] : [state?.colorMode || ColorMode.None],
           autoScale: [true],
-          colorFullBackground: [true],
+          colorFullBackground: [!isTextBasedLanguage],
```

Comment:
If this is because of the possibility of multiple metrics being rendered, then I think it's fine to just ignore it.

Reply:
@flash1293 I did this, because when this is set to true I get this error
<img width="712" alt="image" src="https://user-images.githubusercontent.com/17003240/192259297-5b835251-53d6-4d67-afa3-6f1e80aeb4aa.png">

Any suggestions here?

Reply:
IMHO that's alright for now (especially as it's only the legacy vis)

Reply:
Done https://github.com/elastic/kibana/pull/140469/commits/61b1af125b848214571521610adbeee88c54d9f9